src/main/java/com/expensetracker/mapper/impl/ExpenseMapperImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.expensetracker.mapper.impl;
+
+import com.expensetracker.mapper.ExpenseMapper;
+import com.expensetracker.model.Expense;
+import com.expensetracker.model.ExpensePeriod;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
There should be an annotation called @Mapper

Reply:
The annotation comes from a dependency called org.mapstruct